composer.json

```diff
@@ -18,18 +18,21 @@
         "ext-phar": "*",
         "doctrine/annotations": "^1.10.4",
         "laminas/laminas-coding-standard": "^2.3.0",
-        "laminas/laminas-stdlib": "^3.3.0",
+        "laminas/laminas-stdlib": "^3.6.0",
         "phpunit/phpunit": "^9.5.9",
-        "psalm/plugin-phpunit": "^0.14.0",
-        "vimeo/psalm": "^4.3.1"
+        "psalm/plugin-phpunit": "^0.16.1",
+        "vimeo/psalm": "^4.10.0"
     },
     "suggest": {
         "doctrine/annotations": "Doctrine\\Common\\Annotations >=1.0 for annotation features",
         "laminas/laminas-stdlib": "Laminas\\Stdlib component",
         "laminas/laminas-zendframework-bridge": "A bridge with Zend Framework"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "7.4.99"
```

Comment:
Just curious on why do we need this?

Reply:
If you don't do this, a `composer update` from any `8.0` or `8.1` container (I use docker containers for developing) will cause a dependency bump to something incompatible with PHP 7.4.

That's visible in `git blame`